src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -94,8 +95,10 @@ private void submitInner(FlightContext flightContext)
     final String sqlInsertFlight =
         "INSERT INTO "
             + FLIGHT_TABLE
-            + " (flightId, submit_time, class_name, status, stairway_id, debug_info)"
-            + "VALUES (:flightId, CURRENT_TIMESTAMP, :className, :status, :stairwayId, :debugInfo)";
+            + " (flightId, submit_time, class_name, status, stairway_id, debug_info, "
+            + "    output_parameters_version)"
+            + "VALUES (:flightId, CURRENT_TIMESTAMP, :className, :status, :stairwayId, "
+            + " :debugInfo, :outputParametersVersion)";
```

Comment:
this remindes me the breakage in RBS last time when we added debuginfo into flight context. 
What happens last time is the older flight(in RUNNING or ERROR state) dose not have debugInfo field. But after we upgrade Stairway version and restart server, Stairway tries to recover those old flight, and it is looking for debug info from db(I forgot which table, but might be flightlog table). Then it gets NPE for some of following operations.  It happens a while ago and I forgot the details, just want to make sure we can handle this case gracefully

Reply:
Good point @yonghaoy ...  Both of the new fields are added with default values of 1, so that at database upgrade time existing rows will have this added with version 1.  The version in code is set to 2, so newly written rows will use 2.

I have also created a process for testing database upgrades by storing static data produced with earlier versions of code as static test resources under source control, but I didn't want to add too much to this PR.  That will be a follow-on once this merges.

The tests do use utility methods to make sure upgrade happens at Stairway init time; it would be good to sanity check that "real" Stairway instances will upgrade the database as well.

Reply:
I did a little research, and it looks like all downstream consumers are explicitly configured to do an upgrade migrate at Stairway init time.  

TDR does it by setting `db.migrate.updateAllOnStart=true` in `application.properties`, while the others all set `migrate-upgrade` to `true` in a Stairway config block of `application.yml`:

TDR: https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/resources/application.properties#L12
WSM: https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/src/main/resources/application.yml#L83
Buffer: https://github.com/DataBiosphere/terra-resource-buffer/blob/master/src/main/resources/application.yml#L29
Janitor: https://github.com/DataBiosphere/terra-resource-janitor/blob/master/src/main/resources/application.yml#L30